### PR TITLE
Feat badge

### PR DESCRIPTION
**Pull request by junaidshaikh-js**
Badge component is added 

- Badge on Avatar 
- Badge on Icons 

Font family of the website is changed to Inter.

Live Preview: https://deploy-preview-9--hydrogen-ui.netlify.app/docs/badge

### CODE REVIEW

**Review by FarhanMobashir — COMMENTED** [review]
all good ✅

File: components/badge.css [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+}
+
+/* button sytles  */
+/* TODO remove when button component will be added */
+
+.btn {
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+  outline: none;
```
Comment (AnuragSahu11) [inline components/badge.css:79]: In yesterday's session Arpan told you should not use outline:none use outline:2px solid transparent; outline-offset: 2px;

**Review by AnuragSahu11 — COMMENTED** [review]
Looks Good to me

File: components/badge.css [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  background-color: rgb(27, 168, 27);
+}
+
+.badge-offline {
+  background-color: red;
+}
+
+.badge-num {
+  background-color: #0099ff;
+}
```
Comment (prachi-sahani) [inline components/badge.css]: try using consistent units for colors

File: components/badge.css [hunk 3]
```diff
@@ -0,0 +1,115 @@
+.badge-container {
+  box-sizing: border-box;
+  width: fit-content;
+  height: fit-content;
+  position: relative;
+}
+
+.badge {
+  border: 1px solid white;
```
Comment (stutisk) [inline components/badge.css]: you can try using HEX or RGB method to set color property

File: docs/badge.html [hunk 4]
```diff
@@ -122,7 +122,116 @@ <h5 class="my-1">Components</h5>
...
+              <section>
+                <h2 id="badge-on-icons">Badge on Icons</h2>
+
+                <p>
+                  Number badges on icons is commonly used pattern to display
+                  number counts of notification or any other countable thing.
+                </p>
+
+                <div class="demo-container">
+                  <button class="btn btn-icon btn-icon-lg badge-container">
```
Comment (prachi-sahani) [inline docs/badge.html:216]: "btn-icon-lg" can be changed to "btn-lg" once you create button component

**Review by TejasShekar — APPROVED** [review]
Looks great.

File: components/badge.css (hunk 1 above)
Comment (junaidshaikh-js) [reply components/badge.css:79]: Solved the issue. Thanks.

File: components/badge.css (hunk 3 above)
Comment (junaidshaikh-js) [reply components/badge.css]: Yes, fixed.

File: components/badge.css (hunk 2 above)
Comment (junaidshaikh-js) [reply components/badge.css]: Fixed, thanks!

File: docs/badge.html (hunk 4 above)
Comment (junaidshaikh-js) [reply docs/badge.html:216]: Noted!

**Pull request merged**